src/middlewared/middlewared/plugins/cloud_sync.py

```diff
@@ -638,7 +638,11 @@ class Config:
         Bool("filename_encryption", default=False),
         Str("encryption_password", default=""),
         Str("encryption_salt", default=""),
-        Cron("schedule", required=True),
+        Cron(
+            "schedule",
+            Str('minute', default='00'),
```

Comment:
Everything here uses double quote, i will nitpick and ask for double quotes too :P